### DISCUSSION TITLE
Discussões do Time2Play

### DISCUSSION THREAD

**Discussion by otaviosbampato** [otaviosbampato/Time2Play#40] [Q&A]
## Discussões do Time2Play

![Logo Time2Play](https://github.com/user-attachments/assets/1278a5d2-0f35-480a-a6b2-af3e83de5014)

> [!IMPORTANT]
  > - Essa é uma discussão com visibilidade ***privada***, somente para _desenvolvedores_ e _proprietários_ do Time2Play. 
  > - Prefira por escrever **AQUI** as suas dúvidas referentes ao uso do software, para que possam ser usadas para futuras referências.
  
  * Sinta-se livre para tirar quaisquer dúvidas que possa ter.
  * Compartilhe ideias e sugestões.
  * Mantenha boas práticas de escrita e referenciamento de anexos relevantes à questão.

**Comment by otaviosbampato** [comment]
# Guia do escritor 📝

A fim de manter o hub de discussões organizado e funcional, devemos seguir alguns padrões de escrita e pergunta.

## _O que_ perguntar?

Essa discussão deve ser utilizada para **_todo fim de discussão que concerne ao Time2Play_**. Dentre os possíveis temas de discussão, temos:
* Report de bugs
* Ideias de features
* Sugestões de melhoria
* Críticas e elogios
* Dúvidas gerais
* Etc.

## _Como_ perguntar?
1. Perguntas devem iniciar com um título geral, que elucide o leitor sobre o tema atinente ao post, e preferencialmente cite à qual parte do software se refere. 

* Um bom título seria "**Report de bug durante cadastro de cliente**", e um título ruim seria "**Problema no backend**".

 * Um bom título seria "**Crítica sobre lógica de exclusão de clientes**", e um título ruim seria "**Parte do backend mal-feito**".
 
 * Um bom título seria "**Dúvida sobre implementação na lógica de consulta de reservas**", e um título ruim seria "**Não sei o que esse código faz**".

 2. Após a titulação do post, o mesmo deve conter uma descrição, o mais detalhada possível, que explique como o problema ou a dúvida surgiram, a razão da crítica ou sugestão ou como e por que surgiu a dúvida.  

 3. Procure sempre manter a clareza em sua escrita e tente, ao máximo possível, elucidar de forma completa a questão.  

> [!Tip]
  > Sinta-se livre e encorajado pra inserir o _máximo_ número de anexos que possam explicar a razão por trás do ocorrido, ou da sua linha de raciocínio. Isso ajuda os futuros leitores a encontrar soluções semelhantes para possíveis problemas, e ajuda os desenvolvedores a entender a razão por trás do pensamento.

## Considerações finais

* Esse é um hub _**por**_ desenvolvedores do projeto _**para**_ desenvolvedores do projeto. Portanto, a fim de manter o software fino, sugerimos que **mantenha as notificações ligadas** e **esforce-se para ajudar na área que você melhor entende.**

* É recomendado um esforço para obter um tempo máximo de resposta de 48h em dias úteis.

## Autoria
Este é um parco guia feito manualmente que deve servir somente como _guia de boas práticas_. Sinta-se livre para sugerir modificações para aqui, também.

Escrito por [Otávio Sbampato](https://github.com/otaviosbampato) no dia 16/12/2024.

**Comment by FredMaia** [comment]
Sabe muito!
